.travis.yml

```diff
@@ -17,6 +17,7 @@ install:
   - docker-compose -f ./docker/docker-compose.yml up -d
 # command to run tests
 script:
+    - python setup.py install
```

Comment:
How does this help?

Reply:
The problem was there is no `dorks.pickle` and `user_dorks.pickle` file which is created during `setup`.

 https://github.com/mushorg/tanner/blob/9fa7289896d194e0ca7b03faec3dfd72553271a8/tanner/dorks_manager.py#L59

Reply:
Is this still required?

Reply:
For now its not required, but I think its good to have this.